### DISCUSSION TITLE
The Carla client library (.egg) is tied to specific Python version

### DISCUSSION THREAD

**Discussion by farhanhubble** [carla-simulator/carla#5022] [General]
I was trying to launch the `manual_control.py` script and kept getting the error:

```
ModuleNotFoundError: No module named 'carla'
```

I looked at the script and it turns out that it looks for an egg file with a very specific Python version via:

```Shell
('../carla/dist/carla-*%d.%d-%s.egg' % (
             sys.version_info.major,
             sys.version_info.minor,
             'win-amd64' if os.name == 'nt' else 'linux-x86_64'))

```

I am using v0.9.13 and running Python 3.6. However, the `.egg` bundled with this version are `carla-0.9.13-py3.7-linux-x86_64.egg` and `carla-0.9.13-py2.7-linux-x86_64.egg`. As a hack, I am able to rename the file to match my Python version but I was wondering if the `.egg` file corresponding to Python version 3.6 can be downloaded from somewhere.

Also it would help if this point in mentioned in the docs.

**Comment by ichangxijin** [comment]
pip install carla

**Comment by NicoFrank** [comment]
Did you find a solution? Have the same problem
